src/player/strategy.cpp

```diff
@@ -1320,13 +1320,20 @@ Strategy::getPosition( const int unum ) const
 
     try
     {
-        return M_positions.at( number - 1 );
+        Vector2D pos = M_positions.at(number - 1);
+        Setting &setting = Setting::i();
+        // if aggressive mode just move all players to left
+        double new_pos = pos.x + ((pos.x/10) * ((0.5 - setting.aggressiveness)/0.5));
```

Comment:
the formula does not work correctly
```html
<html>
<body>
<!--StartFragment--><google-sheets-html-origin><style type="text/css"><!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}--></style>
  | 0 | 0.3 | 0.5 | 0.7 | 1
-- | -- | -- | -- | -- | --
-50 | -55 | -52 | -50 | -48 | -45
-40 | -44 | -41.6 | -40 | -38.4 | -36
-30 | -33 | -31.2 | -30 | -28.8 | -27
-20 | -22 | -20.8 | -20 | -19.2 | -18
-10 | -11 | -10.4 | -10 | -9.6 | -9
0 | 0 | 0 | 0 | 0 | 0
10 | 11 | 10.4 | 10 | 9.6 | 9
20 | 22 | 20.8 | 20 | 19.2 | 18
30 | 33 | 31.2 | 30 | 28.8 | 27
40 | 44 | 41.6 | 40 | 38.4 | 36
50 | 55 | 52 | 50 | 48 | 45

<!--EndFragment-->
</body>
</html>
```

x = (x+105) / 5.0 * (t - 0.5) + x - 105
https://docs.google.com/spreadsheets/d/1V0sJgG66LEkwDVnO1EDUsrg_4ASPfBRzCvZt9L7zxzg/edit?usp=sharing